cpp/modules/deck.gl/layers/src/line-layer/line-layer-vertex.glsl.h

```diff
@@ -18,41 +18,228 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-static const char* vs = R"GLSL(
-#define SHADER_NAME line-layer-vertex-shader
+static const char* projectVS = R"GLSL(
```

Comment:
Put this in a separate file project-vertex.glsl.h in this same directory for now and #include it here?